control/control.go

```diff
@@ -94,8 +88,8 @@ func (srv *HTTPServer) Run(bus *events.EventBus) {
 
 // Start sets up API routes with the event bus, listens on the control
 // socket, and serves the HTTP server.
-func (srv *HTTPServer) Start() {
-	endpoints := &Endpoints{srv.Bus}
+func (srv *HTTPServer) Start(bus *events.EventBus) {
+	endpoints := &Endpoints{bus}
```

Comment:
Hopefully we can look at this together tomorrow because I'm unsure what benefit we're getting by this refactor. I think we both had hoped this change set would remove a lot of cruft around the server sections, which I'm not certain of now.